src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -1073,4 +1087,227 @@ public CompactionInfo getCompactionInfo()
         }));
         return new CompactionInfo(redundantBefores, ranges, durableBefore.get());
     }
+
+    @Override
+    public void awaitForTableDrop(TableId id)
```

Comment:
this method name is a bit odd - should be `awaitTableDrop` or `waitForTableDrop`

Reply:
switched to `awaitTableDrop`